src/applications/personalization/view-representative/components/ViewRepresentativeDetails.jsx

```diff
@@ -16,35 +58,66 @@ export const ViewRepresentativeDetails = props => (
+      What can a representative do?
+    </h2>
     <p>
       A recognized representative can file a claim or appeal on your behalf, or
       help you gather supporting documents (like a doctor’s report or medical
       test results). A representative can also provide added support, like help
       getting transportation or emergency funds.
     </p>
-    <p>
-      <strong>
-        Accredited representatives and VSOs can help you understand and apply
-        for VA benefits, like:
-      </strong>
-    </p>
-    <ul>
-      <li>Financial support (monthly payments)</li>
-      <li>Education</li>
-      <li>Vocational Rehabilitation & Employment (VR&E)</li>
-      <li>Home loans</li>
-      <li>Life insurance</li>
-      <li>Pension</li>
-      <li>Health care</li>
-      <li>Burial benefits</li>
-    </ul>
+    <AdditionalInfo triggerText="What are some examples of what a representative can do?">
+      <ul>
+        <li className="vads-u-margin-top--3">
+          Financial support (monthly payments)
+        </li>
+        <li className="vads-u-margin-top--2">Education</li>
+        <li className="vads-u-margin-top--2">
+          Vocational Rehabilitation & Employment (VR&E)
+        </li>
+        <li className="vads-u-margin-top--2">Home loans</li>
+        <li className="vads-u-margin-top--2">Life insurance</li>
+        <li className="vads-u-margin-top--2">Pension</li>
+        <li className="vads-u-margin-top--2">Health care</li>
+        <li className="vads-u-margin-top--2">Burial benefits</li>
+      </ul>
+    </AdditionalInfo>
+
     <p>
       VSOs work on behalf of Veterans and service members, as well as their
       dependents and survivors.{' '}
       <a href="/disability/get-help-filing-claim/">
         Find out more about accredited representatives
       </a>
     </p>
+
+    <h2 id="whatDoesItCost" className="vads-u-font-size--h3">
+      What does it cost to use a recognized representative or a VSO?
+    </h2>
+    <p>
+      No one should charge you a fee to help you file your application for
+      benefits, but they may charge you for unusual expenses. Representatives
+      may charge for their services only after we've made a decision about your
+      original claim. Ask up front what fees you'll be charged. If you believe a
+      claims agent or attourney charged a fee that is too high, you can
+      challenge it.
+    </p>
+    <div className="help-footer-box">
+      <h2
+        className="help-heading vads-u-border-bottom--3px vads-u-border-color--link-default vads-u-font-size--h3 vads-u-padding-bottom--1
+"
+      >
+        Need help?
+      </h2>
+
+      <p className="help-talk">
+        Need help filling out the form or have questions about eligibility?
+        Please call VA Benefits and Services at{' '}
+        <Telephone contact={CONTACTS.VA_BENEFITS} />. If you have hearing loss,
+        call TTY:
+        <Telephone contact={CONTACTS['711']} pattern={PATTERNS['3_DIGIT']} />.
```

Comment:
This text content needs to be updated to what's reflected in the mock up - https://xd.adobe.com/view/e27b7e71-d920-4bcd-9e81-dc62a9665d5e-b728/

`If you have questions about your selected representative....`

Reply:
Updated this